packages/scandipwa/src/component/RecentlyViewedWidget/RecentlyViewedWidget.container.js

```diff
@@ -50,13 +50,12 @@ export class RecentlyViewedWidgetContainer extends PureComponent {
 
     componentDidMount() {
         const {
-            shouldBeUpdated,
             updateRecentViewedProductsInfo,
             recentProducts,
             store
         } = this.props;
 
-        if (shouldBeUpdated && Object.entries(recentProducts).length !== 0) {
```

Comment:
I assume if we remove this check then it will be fetching products each time user visiting page with Recently Viewed Products widget, which is okey, but as I know placeholders are shown when fetching data from BE only if this 'shouldBeUpdated' is true. It is true when user visiting PDP and product is added into localStorage 'RecentlyViewedProducts'. Can we set bool to be true each time we are fetching data for widget?  So placeholders will appear while data will be updating and there will be no mystery appear, for example with price change.

Reply:
Btw, since we fetching data from BE each time user sees widget we can remove logic for setting shouldBeUpdated when adding product into RecentlyViewedProducts in localStorage. Also would be great to rename it, something like 'isLoading' would make more sense.